.github/workflows/backend-integration-test.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       - name: Package Gravitino
         if : ${{ matrix.test-mode == 'deploy' }}
         run: |
-          ./gradlew compileDistribution -x test -PjdkVersion=${{ matrix.java-version }}
+          ./gradlew compileDistribution -x test -PjdkVersion=${{ matrix.java-version }} -PskipDockerDependentTests=false
```

Comment:
Can we shorten the name? It seems too lengthy.

Reply:
how about `withoutDocker`?

Reply:
What about 'skipDockerTests'?